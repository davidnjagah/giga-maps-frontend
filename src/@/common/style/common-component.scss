.side-info-download-api-signup {
  padding: 1.5rem 1rem 1rem 1rem;

  >button {
    width: 100%;
  }
}

.side-info-download-api-avatar-username {
  display: flex;
  align-items: center;
}

.side-info-download-api-avatar {
  display: flex;
  align-items: center;
  justify-content: center;
  width: 2.375rem;
  height: 2.375rem;
  border-radius: 50%;
  background: #277AFF;
  color: #fff;
}

.side-info-download-api-username {
  margin-left: 0.5rem;

  >span {
    color: var(--text-text-primary, #222);
    font-size: 1rem;
    font-style: normal;
    font-weight: 500;
    line-height: 1.375rem;
  }

  >p {
    color: var(--text-text-tertiary, #6F6F6F);
    font-size: 0.75rem;
    font-style: normal;
    font-weight: 400;
    line-height: 1rem;
    letter-spacing: 0.02rem;
  }
}

.side-info-download-api-bottom-line {
  width: 100%;
  height: 0.0625rem;
  background: #E6E6E6;
  margin-top: 0.8rem
}