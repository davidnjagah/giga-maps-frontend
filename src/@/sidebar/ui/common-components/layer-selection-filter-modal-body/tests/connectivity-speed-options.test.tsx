import { render } from '@testing-library/react';
import { testWrapper } from '~/tests/jest-wrapper';
import { PopoverFilterContentSpeed } from '../../styles/layer-filter-modal.style';

describe('Connectivity speed options', () => {
  it('should render component', () => {
    const { asFragment } = render(testWrapper(<PopoverFilterContentSpeed />));
    expect(asFragment).toMatchSnapshot();
  });
});
