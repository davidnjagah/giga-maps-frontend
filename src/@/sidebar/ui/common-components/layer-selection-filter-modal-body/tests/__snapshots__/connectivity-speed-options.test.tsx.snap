// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Connectivity speed options should render component 1`] = `[Function]`;
